Sillycore.EntityFramework/Extensions/AuditEventListener.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.ChangeTracking;
+using Sillycore.Domain.Abstractions;
+
+namespace Sillycore.EntityFramework.Extensions
+{
+    public class AuditEventListener : IEntityEventListener
+    {
+        protected virtual string GetCurrentUserName()
+        {
+            return Thread.CurrentPrincipal?.Identity?.Name;
```

Comment:
This is where you guys were trying to fix but I see no change at all.

Reply:
We thought  that this code can still be valid if this package used in non-web environment. We used our own implementation of IEventListener to get user information in web projects.